### PR TITLE
Stig sle12 initial

### PR DESCRIPTION
**Pull request by abergmann**
#### Description:

This is an initial PR to add a STIG profile for SUSE Linux Enterprise Server 12.

The PR adds the official manual SLES 12 DISA STIG benchmark in version 1 release 2 and then extends the SLE 12 profiles with ```stig.profile``` and the first rule ```installed_OS_is_vendor_supported```.

### CODE REVIEW

**Comment by redhatrises** [comment]
@openscap-ci test this please

File: linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml [hunk 1]
```diff
@@ -29,13 +32,15 @@ severity: high
 
 identifiers:
     cce@rhel8: 80947-5
+    cce@sle12: 83001-8
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml]: @abergmann OOC where did this CCE come from?

Comment (abergmann) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml]: NIST gave us a CCE number block and we started to assign those identifiers to the configuration settings.

Comment (redhatrises) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml]: @abergmann would you be willing to add the cce number block under `shared/references/` like we have done for RHEL?

Comment (redhatrises) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml]: @abergmann ^

This way more people can add CCE's to SLE rules.

Comment (redhatrises) [reply linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_vendor_supported/rule.yml]: Ping @abergmann. Can you provide this? We would really like to get these SLE patches merged!

**Pull request merged**